packages/core/pluggableElementTypes/ViewType.ts

```diff
@@ -18,12 +18,18 @@ export default class ViewType extends PluggableElementBase {
 
   displayTypes: DisplayType[] = []
 
-  constructor(stuff: {
-    name: string
-    ReactComponent: ViewReactComponent
-    stateModel: IAnyModelType
-  }) {
+  extendedName?: string
```

Comment:
could add a short comment possibly

Reply:
added:
> allows for views that are extended from other views to be added as a new view type in the plugin manager

specifically this logic from `plugins/core/PluginManager.ts`:

```js
addViewType(
    creationCallback: (pluginManager: PluginManager) => ViewType,
  ): this {
    const callback = () => {
      const newView = creationCallback(this)
      ;(this.getElementTypesInGroup('display') as DisplayType[]).forEach(
        display => {
          if (
            (display.viewType === newView.name ||
              display.viewType === newView.extendedName) &&
            // view may have already added the displayType in its creationCallback
            !newView.displayTypes.includes(display)
          ) {
            newView.addDisplayType(display)
          }
        },
      )
      return newView
    }
    return this.addElementType('view', callback)
  }
```

since the view names were different but the displays are the same, this is how i "overload" lgv